backend/src/appointment/routes/schedule.py

```diff
@@ -275,7 +302,8 @@ def request_schedule_availability_slot(
 
     # Sending confirmation email to owner
     background_tasks.add_task(
-        send_confirmation_email, url=url, attendee_name=attendee.name, date=date, to=subscriber.preferred_email
+        send_confirmation_email, url=url, attendee_name=attendee.name, attendee_email=attendee.email, date=date,
+        to=subscriber.preferred_email
```

Comment:
This is not really related to this PR, but I found this during testing and decided to just fix it. Somehow we missed providing the attendee email which resulted in an error which was thrown on booking.

Reply:
Thank you!